scalding-core/src/main/scala/com/twitter/scalding/filecache/DistributedCacheFile.scala

```diff
@@ -49,12 +49,34 @@ object URIHasher {
  *
  * {{{
  * class YourJob(args: Args) extends Job(args) {
- *   val theCachedFile = DistributedCacheFile("hdfs://ur-namenode/path/to/your/file.txt")
+ *   val theCachedFile = DistributedCacheFile("/path/to/your/file.txt")
  *
  *   def somethingThatUsesTheCachedFile() {
  *     doSomethingWith(theCachedFile.path) // or theCachedFile.file
  *   }
  * }
+ *
+ * example with Execution:
+ *
+ * {{{
+ * object YourExecJob extends ExecutionApp {
+ *  override def job =
+ *    Execution.withCached("/path/to/your/file.txt") { file =>
+ *      doSomething(theCachedFile.path)
```

Comment:
should this be `doSomething(file.path)` ? 

do we need to give the user a `CachedFile` and make them call `.path` on it, or should we just give them a `new File(cached.path)` directly?